assemble/conf/log4j-service.properties

```diff
@@ -15,42 +15,48 @@
 
 # Write out everything at the DEBUG level to the debug log
 log4j.appender.A2=org.apache.log4j.RollingFileAppender
-log4j.appender.A2.File=${org.apache.accumulo.core.dir.log}/${org.apache.accumulo.core.application}${accumulo.service.instance}_${org.apache.accumulo.core.ip.localhost.hostname}_fromprops.debug.log
-log4j.appender.A2.MaxFileSize=1000MB
-log4j.appender.A2.MaxBackupIndex=10
+log4j.appender.A2.File=${accumulo.log.dir}/${accumulo.application}${accumulo.service.instance}_${accumulo.local.hostname}.debug.log
+log4j.appender.A2.MaxFileSize=100MB
+log4j.appender.A2.MaxBackupIndex=5
 log4j.appender.A2.Threshold=DEBUG
 log4j.appender.A2.layout=org.apache.log4j.PatternLayout
 log4j.appender.A2.layout.ConversionPattern=%d{ISO8601} [%-8c{2}] %-5p: %m%n
 
 # Write out INFO and higher to the regular log
 log4j.appender.A3=org.apache.log4j.RollingFileAppender
-log4j.appender.A3.File=${org.apache.accumulo.core.dir.log}/${org.apache.accumulo.core.application}${accumulo.service.instance}_${org.apache.accumulo.core.ip.localhost.hostname}_fromprops.log
-log4j.appender.A3.MaxFileSize=1000MB
-log4j.appender.A3.MaxBackupIndex=10
+log4j.appender.A3.File=${accumulo.log.dir}/${accumulo.application}${accumulo.service.instance}_${accumulo.local.hostname}.log
+log4j.appender.A3.MaxFileSize=100MB
+log4j.appender.A3.MaxBackupIndex=5
```

Comment:
They were set to keep 20GB (10G for log & 10G for debug.log) of logs for each Accumulo service (tserver, master, gc, etc).  I understand that a user might want to do that on a beefy production node but I don't think it should be the default for users out of the box.  I have run into the problem of disks filling up due to logs when running a SSD VMs in AWS if these defaults are used.  This change will keep up to 1GB of logs per service on each node.